docs/MANUAL.md

```diff
@@ -154,6 +155,16 @@ Oragono can also be configured using environment variables, using the following
 Many administrative actions on an IRC server are performed "in-band" as IRC commands sent from a client. The client in question must be an IRC operator ("oper", "ircop"). The easiest way to become an operator on your new Oragono instance is first to pick a strong, secure password, then "hash" it using the `oragono genpasswd` command (run `oragono genpasswd` from the command line, then enter your password twice), then copy the resulting hash into the `opers` section of your `ircd.yaml` file. Then you can become an operator by issuing the IRC command: `/oper admin mysecretpassword`.
 
 
+## Rehashing
+
+Most configuration settings in Oragono can be changed at runtime by "rehashing", i.e., reloading the configuration file. There are two ways to rehash Oragono:
+
+1. If you are an operator with the `rehash` capability, you can issue the `/REHASH` command (you may have to `/quote rehash`, depending on your client)
+1. You can send the `SIGHUP` signal to Oragono, e.g., via `killall -HUP oragono`
+
+Rehashing also reloads TLS certificates and the MOTD. Some configuration settings cannot be altered by rehash. You can monitor either the response to the `/REHASH` command, or the server logs, to see if your rehash was successful.
```

Comment:
We should mention the 'config options set by environment variables won't be able to be changed' thing here, and maybe in the config files as well near the env-variables option?